src/DocumentsRepository/Utils.ts

```diff
@@ -68,6 +68,7 @@ export function _getErrorResponseParserFn( this:void, registry:DocumentsRegistry
 	return ( error:HTTPError | Error ) => {
 		if( !("response" in error) ) return Promise.reject( error );
 		if( !error.response.data ) return Promise.reject( error );
+		if( error.response.status >= 400 && error.response.status <= 499 ) return Promise.reject( error );
```

Comment:
Since the __addPointer method has been fixed to not throw the IDAlreadyInUseError this line can be removed so that the error sent by the platform is still parsed for the users by the SDK.